### PR TITLE
Submitting psuedocode

### PR DESCRIPTION
**Pull request by cody-berry**

### CODE REVIEW

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # Given a string of uppercase letters, this function removes AB and CD strings as much
+    # as possible and then returns the length of the string. note that removing an AB or
+    # CD string might produce a new one.
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py:4]: Excellent function description! 🍕

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    def minLength(self, s: str) -> int:
+        # what is the index of the first character we are considering?
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py]: Comments shouldn't be questions 👎

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+        # iterate until the index is 2 characters from the length of the string
+
+            # if the pair of characters is AB or CD...
+
+                # remove the characters...
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py:12]: Are you modifying the collection you're iterating? That's very bad practice!

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+                # and if this is not the first index...
+
+                    # decrement the index so that next iteration, we consider whether
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py:16]: Ooo this is clever. I did not think of this...

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+                    # it cleared the way for an AB or CD. 
+
+            # otherwise...
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py:19]: I think you're over complicating this by trying to look at two characters each iteration so you have to handle the case if the 2nd character is an A or C. I think you should just iteration one character each time and do the same check. That would simplify things.

File: 2696. Minimum String Length After Removing Substrings/solution.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+                # if the next character is A or C...
+
+                    # we can consider it...
+
+                # and otherwise...
+
+                    # skip it and advance twice.
+
+        return 2
```
Comment (aeruru) [inline 2696. Minimum String Length After Removing Substrings/solution.py]: You forgot to comment what you're going to return

File: 2696. Minimum String Length After Removing Substrings/solution.py (hunk 4 above)
Comment (cody-berry) [reply 2696. Minimum String Length After Removing Substrings/solution.py:16]: For some reason I can't resolve this comment.

Comment (cody-berry) [reply 2696. Minimum String Length After Removing Substrings/solution.py:16]: I meant "this conversation."

**Pull request merged**